UnityProject/Assets/MainApp.androidlib/build.gradle

```diff
@@ -5,13 +5,12 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.0.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
```

Comment:
Could you integrate build.gradle changes from https://github.com/Unity-Technologies/uaal-example/pull/104/files, since this PR has landed - https://github.cds.internal.unity3d.com/unity/unity/pull/47453, this will ensure this module uses same targetSdk/minSdk and others as set in Unity